src/system_metrics/mongoose_system_metrics_collector.erl

```diff
@@ -215,3 +216,11 @@ calculate_stanza_rate(PrevReport, NewCount) ->
             undefined -> Count;
             Total -> Count-Total
         end} || {Type, Count} <- NewCount].
+
+get_config_type() ->
+    ConfigPath = ejabberd_config:get_config_path(),
+    ConfigType = case re:run(ConfigPath, ".*toml") of
```

Comment:
```> re:run("i-dont-like-toml-so-i-use.cfg", ".*toml").
{match,[{0,16}]}```

Could you use `filename:extension/1` like `mongoose_config_parser:parse_file/1`?

Reply:
sure, thanks for the hint.